src/adapters/EOFeedRegistryAdapterBase.sol

```diff
@@ -114,6 +120,20 @@ abstract contract EOFeedRegistryAdapterBase is OwnableUpgradeable, EOFeedFactory
         return IEOFeedAdapter(feedAdapter);
     }
 
+    /**
+     * @notice Remove the feedAdapter
+     * @param base The base asset address
+     * @param quote The quote asset address
+     */
+    function removeFeedAdapter(address base, address quote) external onlyOwner {
```

Comment:
isn't it simpler to provide the feedId as a parameter instead of resolving it? or we have to get it for cleanup? because I don't see we dtore anywhere the base and quote other than keys in the mapping..